app/UI/sign-in.tsx

```diff
@@ -26,15 +25,10 @@ interface State {
         type: "signin";
         private_key: string;
     } | {
-        type: "name";
-        name: string;
-    } | {
-        type: "backup";
-        name: string;
+        type: "create-key";
```

Comment:
1. remove `name` step
2. rename `backup` to `create-key` to better describe the intention